conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -433,11 +365,8 @@ private void trackAdEnd() {
         }
         adStarted = false;
 
-        try {
-            client.adEnd(sessionId);
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
-        }
+        Log.d(TAG, "Report ad break ended");
+        convivaVideoAnalytics.reportAdBreakEnded();
```

Comment:
Are there separate APIs for reporting AdBreakEnd and AdEnd? The trackAdEnd() method is being called when AdEnd event is received from Bitmovin player instead of when AdBreakEnd is received. If there are separate APIs, then we need to align the calls with respective Ad events from player.

Reply:
It's unfortunate that they are called different things, but they are the only calls available and actually lead to more detailed reporting than the previous version.

Reply:
ok, thanks.